subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -63,3 +414,43 @@ You can enable this behavior by using the `enforcedPlatform` keyword, instead of
 include::sample[dir="snippets/dependencyManagement/managingTransitiveDependencies-enforcedConstraintsFromBOM/groovy",files="build.gradle[tags=dependency-on-bom]"]
 include::sample[dir="snippets/dependencyManagement/managingTransitiveDependencies-enforcedConstraintsFromBOM/kotlin",files="build.gradle.kts[tags=dependency-on-bom]"]
 ====
+
+[[sub:platforms-vs-catalog]]
+== Should I use a platform or a catalog?
+
+Because platforms and catalogs both talk about dependency versions, can both be used to share dependency versions in a project, there might be a confusion regarding what to use and if one is preferable to the other.
+
+Ignoring the incubating state of version catalogs, the answer is that you should always use catalogs and _maybe_ use platforms in addition.
```

Comment:
```suggestion
The answer is that you should always use catalogs and _maybe_ use platforms in addition.
```

We will forget to remove this and in any case it does not help the user to read this here.